sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceAsyncClient.java

```diff
@@ -329,6 +331,72 @@ public String getAccountUrl() {
                 null, Context.NONE), timeout);
     }
 
+
+    /**
+     * Returns a reactive Publisher emitting the blobs in this account whose tags match the query expression. For more
+     * information, including information on the query syntax, see the <a href="DOC LINK NEEDED">Azure Docs</a>.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.blob.BlobServiceAsyncClient.filterBlobs#String}
+     *
+     * @param query Filters the results to return only blobs whose tags match the specified expression.
+     * @return A reactive response emitting the list of blobs.
+     */
+    public PagedFlux<FilterBlobItem> findBlobsByTags(String query) {
```

Comment:
Thoughts on adding some convenience method to build a query?

Reply:
I think this is a part of an ongoing email thread about what sort of object model we want to offer, if any, to support building queries.